inventory/byo/hosts.example

```diff
@@ -99,6 +99,9 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # set RPM version for debugging purposes
 #openshift_pkg_version=-3.0.0.0
 
+# Configure custom master certificates
+#openshift_master_named_certificates=[{"certfile": "/path/to/custom1.crt", "keyfile": "/path/to/custom1.key"}]
```

Comment:
I would let them set a `names:[...]` array for each cert, and only do the autodetect if that is missing. That gives you an escape hatch if they don't want to use all the SANs in the cert, or use the cert for a name that is not a SAN